### PR TITLE
Consistent styling for email confirmation pages

### LINKED ISSUE #577: Improve design of email confirmation page
Looks a bit lopsided on a desktop.

Context: People see this page a lot, when they submit a report and ask for email confirmation (instead of supplying a password).

screen shot 2015-09-01 at 15 31 05

### PR DESCRIPTION
**Pull request by zarino**
This PR makes the "Please check your email" pages in the `fixmystreet` cobrand look the same as the other "success" pages, like the one you'll see immediately after clicking the link in your email!

This fixes #577.

It's based on top of https://github.com/mysociety/fixmystreet/tree/1209-right-to-left because that includes some handy confirmation page test endpoints that haven't been merged into master yet.

Here's how they look on desktop, in the `fixmystreet.com` cobrand:

screen shot 2015-09-15 at 17 18 05

screen shot 2015-09-15 at 17 18 09

And in a random selection of other cobrands:

screen shot 2015-09-15 at 17 22 29

screen shot 2015-09-15 at 17 22 42

screen shot 2015-09-15 at 17 23 08

### CODE REVIEW

File: web/cobrands/sass/_layout.scss [hunk 1]
```diff
@@ -498,6 +498,18 @@ body.twothirdswidthpage {
   }
 }
 
+// Centre the login and password change pages,
+// but keep them narrow to match the report sidebar
+body.authpage {
+  .content {
+    margin-left: auto;
```
Comment (dracos) [inline web/cobrands/sass/_layout.scss]: For consistently, even when left/right have been the same in the stylesheet, as here, I have used `#{$left}` and `#{$right}` (which makes it much easier to search for any cases I missed :) ).

**Comment by dracos** [comment]
Whilst the around/report_import test failures are due to unconnected postcode changes (hopefully fixed by #1214), the changes of this PR have, due to text changes looks like, broken some tests in alert_new.t and report_updates.t it would be good to fix. Plus the tick in one of the screenshots I mentioned in IRC. Once they're sorted I'll look at this properly on Thursday, looks good :)

**Comment by zarino** [comment]
@dracos – The missing icon on MakeMyIsland turned out to be down to `bin/make_css` not following symlinks (I have the makemyisland theme directory set up as a symlink in `web/cobrands`).

I've fixed the obvious test failures, and run the tests locally in my VM to check – there's still a bunch of failures, but none of them appear to be due to changes in this branch. Let's see what Travis makes of it…

**Comment by dracos** [comment]
(Aside: I wouldn't put the issue number in every commit on a feature branch, because it can make the issue page a bit confusing #577, and those references seem to stick around for quite some time, multiplying with each rebase. I would leave the reference to one main commit, or the GitHub PR, or add it at the end before the final merge.)

File: templates/web/base/auth/token.html [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
+[%# Shown whenever there's an invalid token, eg while confirming a report or logging in without a password %]
 
-<h1>[% loc('Error') %]</h1>
+  <div class="confirmation-header confirmation-header--failure">
 
-<p>
-    [% loc('We have not been able to confirm your account - sorry.') %]
-    [% loc('This may be because the link is too old or already used, or the address was not copied correctly.') %]
-</p>
+    <h1>[% loc('Sorry, that was&rsquo;t a valid link') %]</h1>
```
Comment (dracos) [inline templates/web/base/auth/token.html]: Typo in this line.

**Comment by dracos** [comment]
Merged to master, thank!

**Pull request closed without merging**